go/vt/schema/online_ddl.go

```diff
@@ -84,19 +83,20 @@ const (
 
 // OnlineDDL encapsulates the relevant information in an online schema change request
 type OnlineDDL struct {
-	Keyspace         string          `json:"keyspace,omitempty"`
-	Table            string          `json:"table,omitempty"`
-	Schema           string          `json:"schema,omitempty"`
-	SQL              string          `json:"sql,omitempty"`
-	UUID             string          `json:"uuid,omitempty"`
-	Strategy         DDLStrategy     `json:"strategy,omitempty"`
-	Options          string          `json:"options,omitempty"`
-	RequestTime      int64           `json:"time_created,omitempty"`
-	MigrationContext string          `json:"context,omitempty"`
-	Status           OnlineDDLStatus `json:"status,omitempty"`
-	TabletAlias      string          `json:"tablet,omitempty"`
-	Retries          int64           `json:"retries,omitempty"`
-	ReadyToComplete  int64           `json:"ready_to_complete,omitempty"`
+	Keyspace string      `json:"keyspace,omitempty"`
+	Table    string      `json:"table,omitempty"`
+	Schema   string      `json:"schema,omitempty"`
+	SQL      string      `json:"sql,omitempty"`
+	UUID     string      `json:"uuid,omitempty"`
+	Strategy DDLStrategy `json:"strategy,omitempty"`
+	Options  string      `json:"options,omitempty"`
+	// Stateful fields:
+	MigrationContext   string          `json:"context,omitempty"`
+	Status             OnlineDDLStatus `json:"status,omitempty"`
+	TabletAlias        string          `json:"tablet,omitempty"`
+	Retries            int64           `json:"retries,omitempty"`
+	ReadyToComplete    int64           `json:"ready_to_complete,omitempty"`
+	WasReadyToComplete int64           `json:"was_ready_to_complete,omitempty"`
```

Comment:
Was going to ask if we can't use `atomic.Int64` here (and for others where appropriate), but then found https://github.com/golang/go/issues/54582.